base/ubi/9.4-minimal/almalinux.repo

```diff
@@ -0,0 +1,6 @@
+[base-9]
```

Comment:
```suggestion
[base-9]
```
when using `--enablerepo base-9` it might need a bit more specificity, like `alma-base-9`